documentation/modules/rn-2.2.adoc

```diff
@@ -94,3 +94,37 @@ The error status message for a VM with no operating system on the *Migration pla
 .Network, storage, and VM referenced by name in the Plan CR are not displayed in the web console.
 
 If a Plan CR references storage, network, or VMs by name instead of by ID, the resources do not appear in the {project-short} web console. The migration plan cannot be edited or duplicated. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1986020[*BZ#1986020*])
+
+.Restart of the CatalogSource pod is required to upgrade MTV 2.1.0 to MTV 2.2.0.
+
+To restart the CatalogSource pod, do the following:
+
+.  In the *Subscriptions* tab of the {operator-name} in the {ocp-short} web console, take note of the name in the *CatalogSource* field.
+.  In the CLI, run the following command:
++
+[source,terminal]
+----
+  $ oc get pod -n openshift-marketplace
```

Comment:
@mguetta1 Is "openshift-marketplace" the correct namespace for downstream?

Reply:
@apinnick yes. It is the namespace where the catalog source CRs and their pods are located